frontend/src/metabase/search/components/filters/CreatedByFilter/CreatedByContent.tsx

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable react/prop-types */
+import { useState } from "react";
+import { isEqual } from "underscore";
+import { t } from "ttag";
+import { useUserListQuery } from "metabase/common/hooks/use-user-list-query";
+import type { UserListResult } from "metabase-types/api";
+import { Center, Loader, Text, TextInput } from "metabase/ui";
+import {
+  CreatedByContainer,
+  CreatedByContentContainer,
+} from "metabase/search/components/filters/CreatedByFilter/CreatedByContent.styled";
+import type { SearchSidebarFilterComponent } from "metabase/search/types";
+import { UserListElement } from "metabase/search/components/filters/CreatedByFilter/UserListElement";
+
+export const CreatedByContent: SearchSidebarFilterComponent<"created_by">["ContentComponent"] =
+  ({ value, onChange }) => {
+    const { data: users = [], isLoading } = useUserListQuery();
+    const [userFilter, setUserFilter] = useState("");
+
+    const filteredUsers = users.filter(user =>
+      user.common_name.toLowerCase().includes(userFilter.toLowerCase()),
+    );
+
+    const onUserSelect = (user: UserListResult) => {
+      if (value && value.length > 0 && isEqual(value[0], String(user.id))) {
+        onChange([]);
+      } else {
+        onChange([String(user.id)]);
+      }
+    };
+
+    return isLoading ? (
+      <Center>
+        <Loader data-testid="loading-spinner" />
+      </Center>
+    ) : (
+      <CreatedByContainer p="md" h="100%" spacing="xs">
+        <TextInput
```

Comment:
It feels like there should be a placeholder of some sort, It's difficult to tell what this input is for on a first glance.